lib/ansible/plugins/doc_fragments/action_common_attributes.py

```diff
@@ -11,84 +10,58 @@ class ModuleDocFragment(object):
     # Standard documentation fragment
     DOCUMENTATION = r'''
 attributes:
-    action:
-      description: Indicates this has a corresponding action plugin so some parts of the options can be executed on the controller
-      support: none
+    check_mode:
+      description: Can run in check_mode and return changed status prediction withought modifying target
+    diff_mode:
+      description: Will return details on what has changed (or possibly needs changing in check_mode), when in diff mode
+    platform:
+      description: Target OS/families that can be operated against
+'''
+
+    actiongroups = r'''
```

Comment:
The additional fragments in here need to get backported to make the compatibility shims in stable-2.9, stable-2.10 and stable-2.11 working properly.

Reply:
yes, once this is merged and we have finalized it, I'll create backport version (still empty, just declaring all vars used)